design/routing-design.md

```diff
@@ -123,6 +123,34 @@ Exact Path based routing makes decisions based upon the path of the request (e.g
 Wildcard path based routing allows for a portion of the path to be dynamic (e.g. /path/*/foo).
 This is useful to allow for many types of request paths to be routed to the same backed without specifying them explicitly up front.
 
+- Wildcards defined cannot be the end of the path (e.g. `/foo/*/bar` or `/*/foo` not `/foo*`)
+- Similar to path prefix matching, wildcard path routing requires the condition to start with a `/`
+- There is no change to the sorting of paths. The path `/api/*/foo` would still sort before prefix: `/api/v2`
+- Merging conditions together is allowed. If users define a prefix condition of `/foo` and include that to another Proxy which defines a wildcard path of `/*/bar`, the result is a regex expression of `/foo/*/bar`
```

Comment:
suggestion: leave the implementation details out of the business logic for merging rules. For this part of the document lets lock down the merging rules of different conditions.